webapp/sources/rudder/rudder-rest/src/main/scala/com/normation/rudder/rest/EndpointsDefinition.scala

```diff
@@ -801,6 +800,18 @@ object InfoApi extends ApiModuleProvider[InfoApi] {
   def endpoints = ca.mrvisser.sealerate.values[InfoApi].toList.sortBy( _.z )
 }
 
+sealed trait HookApi extends EndpointSchema with InternalApi with SortIndex {
+  override def dataContainer = None // nothing normalized here ?
+}
+object HookApi extends ApiModuleProvider[HookApi] {
+  final case object GetHooks extends HookApi with ZeroParam with StartsAtVersion10 with SortIndex { val z = implicitly[Line].value
```

Comment:
API version 10, 16 or 15 ? @VinceMacBuche

Reply:
not 10, a new version, I guess 16